ToyBox/classes/MainUI/EnhancedUI/PhatLoot.cs

```diff
@@ -276,7 +290,7 @@ public class PhatLoot {
                            }
                        }
                    },
-                   () => {});
+                   () => { });
```

Comment:
Auto formatter made this and the trailing whitespace change. Since I had to touch the file anyway, I think it's worth leaving alone so it doesn't happen in the future.